### PR TITLE
Remove use of deprecated set-output

### PR DESCRIPTION
**Pull request by penx**
### Description

Remove use of set-output from GitHub actions, which has been deprecated

https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -12,15 +12,15 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Read Node version
-        run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
```
Comment (penx) [inline .github/workflows/build.yml:15]: not sure why this was a hash before...

Comment (penx) [reply .github/workflows/build.yml:15]: and not `::set-output`

Comment (andy-hook) [reply .github/workflows/build.yml:15]: I'm not sure either, perhaps we should also replace the separate read and set with the approach from `publish-rc`?

Comment (penx) [reply .github/workflows/build.yml:15]: I think publish-rc is doing the same thing, it has 2 steps (read the nvmrc followed by use actions/setup-node) just the second step in publish-rc is not named.

Comment (andy-hook) [reply .github/workflows/build.yml:15]: Should have been clearer, I meant align them on the naming as they perform the same task. The latter also uses the `v2` action.

Comment (penx) [reply .github/workflows/build.yml:15]: Sure, will take a look later

Have you considered adding your actions to dependabot? Though I think has security implications if you use 3rd party actions.

https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot

Comment (andy-hook) [reply .github/workflows/build.yml:15]: Thanks mate.

> Have you considered adding your actions to dependabot?

  To be honest we haven’t invested much time in our CI config as it has been doing the job we've needed so far, these are good suggestions though.

Comment (penx) [reply .github/workflows/build.yml:15]: @andy-hook I've brought them in line with each other, hopefully what you were wanting :)

**Review by andy-hook — APPROVED** [review]
Perfect, thanks @penx !

**Pull request merged**